bodhi/server/util.py

```diff
@@ -604,7 +604,10 @@ def cmd(cmd, cwd=None):
     if out:
         log.debug(out)
     if err:
-        log.error(err)
+        if p.returncode == 0:
+            log.info(err)
+        else:
+            log.error(err)
```

Comment:
Hello @crungehottman!

I would expect that you would be able to use [```mock```](http://www.voidspace.org.uk/python/mock/index.html) to force ```subprocess.Popen()``` to return a ```MagicMock``` that does what you want for each test. Here's a toy example that shows how I can use ```mock``` to force a fake ```subprocess``` call and make it give the return values and exit code I want for testing:

```
import subprocess                                                                                                    
                                                                                                                     
import mock


def x():
    p = subprocess.Popen(['/bin/echo', '"Hello World!"'], stdout=subprocess.PIPE,
                         stderr=subprocess.PIPE)
    out, err = p.communicate()

    if out:
        print "stdout: {}".format(out)
    if err:
        if p.returncode == 0:
            print "info: {}".format(err)
        else:
            print "ERROR: {}".format(err)


with mock.patch('subprocess.Popen') as Popen:
    Popen.return_value.communicate.return_value = ('yooo', 'sup')
    Popen.return_value.returncode = 1

    x()
```

If you run that, you'll see this output:

```
stdout: yooo
ERROR: sup
```

Then you can basically make the three tests I suggested by messing with the return value and exit codes. By the way, mock can be done via the with statement or through a decorator. If you ```git grep mock``` in the bodhi codebase, you'll see lots of examples of how to do it as a decorator on a test method.

Does that help answer your question?